python/cudf/cudf/utils/hash_vocab_utils.py

```diff
@@ -108,11 +108,11 @@ def _find_hash_for_internal(hash_bin):
             return bins, a, b
 
 
-def _perfect_hash(integers, max_constant):
+def _perfect_hash(integers, rng, max_constant):
```

Comment:
*nit*: Why not the last argument as for most of the other functions that are now accepting an rng?